src/components/core/AppSideBar.vue

```diff
@@ -38,7 +38,12 @@ import {
 } from 'vuex'
 
 import DisplayName from '../util/DisplayName'
-import PrepaidTransactionsControl from '../PrepaidTransactionsControl'
+
+// for some reason, eslint can't handle this file... it loads just fine but
+//  eslint throws a bunch of weird errors so we'll just disable this line
+//
+// eslint-disable-next-line
+import CODXBalanceControl from '../CODXBalanceControl'
```

Comment:
This is weird. What happens if you uncomment out the markup in that file?

Reply:
Huh, I removed the comment and now it appears fine. I did restart my computer this morning so maybe that had something to do with it. Eslint was complaining about the casing of the file not matching the import, even though it definitely matched.